db/migrate/20231108110923_add_gias_group.rb

```diff
@@ -0,0 +1,17 @@
+class AddGiasGroup < ActiveRecord::Migration[7.0]
+  def change
+    create_table :gias_groups, id: :uuid do |t|
```

Comment:
It's possibly not on the card, but can we add the `Unique group identifier (UID)` as I think this the GIAS ID for the record whereas `group_identifier` is from KIM and `UKPRN` is from an external body so possibly not as robust.

I would possibly label the column `urn` as that makes it equivilent to URN on Establishment - not sure why they chose a different name for the value, but it looks like the value is used in GIAS:

https://get-information-schools.service.gov.uk/Groups/Group/Details/2073

Reply:
I can certainly do that - are we sure about calling it URN? Only I'm thinking of the `urn_validator` which expects 6 digits, while these are 4 digits. I know we don't have to use the validator, but it would be confusing for URNs to be 6 digits (and enforced as 6) in one place but 4 here.

Reply:
Fair point - can we still include it, I think it is important `unique_group_identifier` is fine! 👍